spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -35,11 +35,12 @@ A full list of changes delivered in the 2.0 release can be found at link:https:/
 
 - OSGi dependencies marked as "provided" (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/132[#132])
 
-=== API/SPI Changes 
+=== API/SPI Changes
 - ReactiveStreamsFactory.fromSubscriber parameter type change (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/134[#134])
 
 === Other Changes
 - Update to Jakarta EE8 APIs for MP 4.0 (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/128[#128])
 - Javadoc clarification: (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/135[#135]) (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/137[#137])(link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/142[#142]) (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/129[#129])
 - TCK clarification: (link:https://github.com/eclipse/microprofile-reactive-streams-operators/issues/141[#141])
+- Loading ReactiveStreamsEngine and ReactiveStreamsFactory should also work running with a security manager (link:https://github.com/eclipse/microprofile-reactive-streams-operators/pull/170[#170])
```

Comment:
You should add a heading Other Changes on the release notes 3.0 and put this line 45 over there. Please don't update the previous release notes.

Reply:
@Emily-Jiang - is it fine now?